.github/workflows/publish-swift.yml

```
@@ -0,0 +1,140 @@
+        description: 'token to publish to cocoapods'
+        required: true
+
+jobs:
+  build-tag-release:
+    runs-on: macos-latest
+    steps:
+      - name: Checkout breez-sdk-swift repo
+        uses: actions/checkout@v3
+        with:
+          repository: breez/breez-sdk-swift
+          ssh-key: ${{ secrets.REPO_SSH_KEY }}
+          path: breez-sdk-swift
+
+      - name: Checkout breez-sdk repo
+        uses: actions/checkout@v3
+        with:
+          repository: ${{ inputs.repository || github.repository }}
+          ref: ${{ inputs.ref || github.sha }}
+          path: breez-sdk
+
+      - uses: actions/download-artifact@v4
+        with:
+          name: bindings-swift
+          path: breez-sdk/libs/sdk-bindings/bindings-swift/Sources/BreezSDK/
+
+      - uses: actions/download-artifact@v4
+        with:
+          name: sdk-bindings-aarch64-apple-ios
+          path: breez-sdk/libs/target/aarch64-apple-ios/release/
+
+      - uses: actions/download-artifact@v4
+        with:
+          name: sdk-bindings-ios-universal-sim
+          path: breez-sdk/libs/target/ios-universal-sim/release/
+
+      - uses: actions/download-artifact@v4
+        with:
+          name: sdk-bindings-darwin-universal
+          path: breez-sdk/libs/target/darwin-universal/release/
+
+      - name: Create XCFramework
+        working-directory: breez-sdk/libs/sdk-bindings
+        run: |
+          cp bindings-swift/Sources/BreezSDK/breez_sdkFFI.h bindings-swift/breez_sdkFFI.xcframework/ios-arm64/breez_sdkFFI.framework/Headers
+          cp bindings-swift/Sources/BreezSDK/breez_sdkFFI.h bindings-swift/breez_sdkFFI.xcframework/ios-arm64_x86_64-simulator/breez_sdkFFI.framework/Headers
+          cp bindings-swift/Sources/BreezSDK/breez_sdkFFI.h bindings-swift/breez_sdkFFI.xcframework/macos-arm64_x86_64/breez_sdkFFI.framework/Headers
+          cp ../target/aarch64-apple-ios/release/libbreez_sdk_bindings.a bindings-swift/breez_sdkFFI.xcframework/ios-arm64/breez_sdkFFI.framework/breez_sdkFFI
+          cp ../target/ios-universal-sim/release/libbreez_sdk_bindings.a bindings-swift/breez_sdkFFI.xcframework/ios-arm64_x86_64-simulator/breez_sdkFFI.framework/breez_sdkFFI
+          cp ../target/darwin-universal/release/libbreez_sdk_bindings.a bindings-swift/breez_sdkFFI.xcframework/macos-arm64_x86_64/breez_sdkFFI.framework/breez_sdkFFI
+          rm bindings-swift/Sources/BreezSDK/breez_sdkFFI.h
+          rm bindings-swift/Sources/BreezSDK/breez_sdkFFI.modulemap
+
+      - name: Compress xcframework
+        working-directory: breez-sdk/libs/sdk-bindings/bindings-swift
+        run: |
+          zip -9 -r breez_sdkFFI.xcframework.zip breez_sdkFFI.xcframework
+          echo "XCF_CHECKSUM=`swift package compute-checksum breez_sdkFFI.xcframework.zip`" >> $GITHUB_ENV
+
+      - name: Archive xcframework
+        uses: actions/upload-artifact@v4
+        with:
+          name: breez_sdkFFI-${{ inputs.package-version || intouts.ref }}.xcframework
```

This is causing the publish CI to fail when testing
```suggestion
          name: breez_sdkFFI-${{ inputs.package-version || input.ref }}.xcframework
```

---

Damn, typo! Thanks for catching!